paladin/mod.ts

```diff
@@ -73,116 +62,100 @@ export function Paladin(
     // If there is a response, then we know this is occurring after the request
     if (response) {
       // Set "X-XSS-Protection" header. See https://helmetjs.github.io/docs/xss-filter/
-      if (configs["X-XSS-Protection"] !== false) {
+      // if user explicitely enabled it, or left it
+      if (
+        configs["X-XSS-Protection"] === true ||
```

Comment:
why can't this just check for existence? `if (configs["X-XSS-Protection"])`

Reply:
Because if the user does `const csrf = CSRF()`, then `configs["..."]` is undefined

The above is checking:

1.  If the user has explicitly set it to true, eg `CSRF({ "...": true })`
2. And if the user has NOT set it to false
3. Then enable it

Plus, when working with booleans, i prefer to use `if (the var !== true)` as opposed to `if (!the var)`, just because i usually see `if (!something)` as the var being null or undefined, whereas using `if (the var is a boolean value)` is more explicit to what we're trying to check

Reply:
We could check `config` before entering these `if` blocks and use `!!` operator but not a hill I'm dying on